docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -3442,6 +3451,161 @@ the `vrt2` feature. The default vertical advance for these glyphs is the
 horizontal advance of their corresponding target (upright) glyphs. These values
 will also be overridden by `VertAdvanceY` values.
 
+<a name="9.i"></a>
+### 9.i. STAT table
+STAT table is organized as followas:
+
+```fea
+table STAT {
+    # Elided fallback name
+    # Design axes
+    # Axis values
+} STAT;
+```
+
+#### Elided fallback name
+Elided fallback name must be defined and at most once. It can be
+defined using full name entries or an existing name ID.
+
+```fea
+ElidedFallbackName {
+    name <name spec>;+
+};
+```
+
+The syntax for the individual name string entries is similar to that of the
+name table `nameID` entries (see see §[9.e](#9.e)) - the only difference is
+that the introductory keyword is `name`, and the name ID value is omitted,
+since the nameID value is auto-generated by the feature compiler.
+
+Alternatively if the name string matches one of the pre-existing names in the
+`name` table, the name ID can be specified directly and it must be be present
+in the `name` table.
+
+```fea
+ElidedFallbackNameID <name ID>;
+```
+
+#### Design axes
+There can one or more design axes specified in the `STAT` table.
+
+```fea
+DesignAxis <axisTag> <axisOrdering num> {
+    name <name spec>;+
+};+
+```
+
+`axisTag` is a four-letter tag. `axisOrdering` is a signed number.
+
+The syntax for individual name string entries is the same as above.
+
+#### Axis values
+There can one or more axis values specified in the `STAT` table.
+
+```fea
+AxisValue {
+    flag <FLAG>+ ;
+    name <name string>;+
+    location ...;+
+};+
+```
+
+`flag` is optional, and defaults to 0 when omitted. Possible flags are
+`ElidableAxisValueName` and `OlderSiblingFontAttribute`, and more than one flag
+can be specified, e.g:
+
+
+```fea
+flag ElidableAxisValueName OlderSiblingFontAttribute;
+```
+
+There are several formats for `location` statement:
+
+##### location format A
+```fea
+location <axisTag> <value>;+
+```
+
+`axisTag` is a 4-letter tag and must correspond to one of the defined design
+axes in the table. `value` is signed number and can be specified using decimal,
+hex and octal formats as well (see §[9.e](#9.e)).
+
+There can be more than one `location` statement when this format is used (in
+this case the axis value will be `STAT` format 4 `AxisValue`, and format 1
+otherwise).
+
+##### location format B
+```fea
+location <axisTag> <nominalValue> <rangeMinValue> - <rangeMaxValue>;
+```
+
+Format for `axisTag` and the other values as above, and there must be space
```

Comment:
Should this say "there must be a space before and after the `-` to distinguish it from a possible minus sign for `<rangeMaxValue>`" ?

Reply:
The grammar does not require space after the number, but requires space before the hyphen otherwise it is would be considered a minus sign.